Lib/defconQt/controls/glyphCellView.py

```diff
@@ -274,8 +275,7 @@ def paintEvent(self, event):
 
             if visibleRect.intersects(visibleRect.__class__(*rect)):
                 pixmap = self._getCurrentRepresentation(glyph)
-                painter.drawPixmap(
-                    left, t, pixmap, 0, 0, cellWidth, cellHeight)
```

Comment:
This was clipping `pixmap` to the unscaled width and height, I should have multiplied the `cellWidth` and `cellWidth` by `devicePixelRatio`, but it is simpler not to pass any width or height at all since we are creating the pixmap at the exact cell width and height already.